geometric_params:
  # Units: [m, deg]
  r_c: 0.145
  ang_b_m: 54.735610
  r_b: 0.20506096413551428
  r_s_out: 0.2511473651308862

  d: 0.05939697
  r_e: 0.05939697
  r_s_in: 0.08400000053786785